e2e/connect_nodes_test/test_multiple_connections/test.yml

```diff
@@ -0,0 +1,11 @@
+query: >
+    MATCH (n)
+    WITH collect(n) as nodes
+    CALL graph_util.connect_nodes(nodes) YIELD connections
+    UNWIND connections AS connection
+    RETURN connection.id AS id;
```

Comment:
Is this part guaranteed that it will always return 1,2,3? Maybe add order by part?

Reply:
Yes, I put it like that because of the IDs that get assigned in the database and then get traversed like that in the query module because of api methods?